tauri-update/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "tauri-update"
+version = "0.2.0"
+authors = ["Lucas Fernandes Gonçalves Nogueira <lucas@quasar.dev>", "Daniel Thompson-Yvetot <denjell@sfosc.org>", "Tensor Programming <tensordeveloper@gmail.com>"]
+license = "MIT"
+homepage = "https://tauri-apps.org"
+repository = "https://github.com/tauri-apps/tauri"
+description = "Make tiny, secure apps for all desktop platforms with Tauri"
+edition = "2018"
+exclude = ["test/fixture/**"]
+
+[dependencies]
+reqwest = "0.9"
```

Comment:
Possibly out of scope, but I thought we switched away from reqwest? If this is going to be pre-built before our users get it, I'm not so concerned

Reply:
Doesn't build because the updater is effectively just dead code. It isn't being used by us at the moment because we need to redesign it entirely.  So I spun it into its own crate to keep it out of the way.

Reply:
Alrighty